test/features/support/PoolEnv.js

```diff
@@ -386,8 +390,8 @@ function PoolEnv() {
 
   //   for (let userIndex = 1; userIndex < 3; userIndex++) {
   //     let wallet = await this.wallet(userIndex)
-  //     let chance = await this.env.ticket.chanceOf(wallet._address)
-  //     console.log(chalk.green(`User ${userIndex} (${wallet._address}) chances: ${ethers.utils.formatEther(chance)}`))
+  //     let chance = await this.env.ticket.chanceOf(wallet.address)
+  //     console.log(chalk.green(`User ${userIndex} (${wallet.address}) chances: ${ethers.utils.formatEther(chance)}`))
```

Comment:
✂️  is this fxn even needed anymore?  whats happening here?

Reply:
I dont know! This was already commented out. I've removed it.